Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Properties/ParameterExtractor.cs

```diff
@@ -218,6 +219,13 @@ ILogger<ParameterExtractor> logger
           return null;
         }
 
+        // NOTE: first check if this is a "Type ID" parameter, if so return the actual ID
+        // explicit (aka hacky) check here related to [CNX-1125](https://linear.app/speckle/issue/CNX-1125/publish-type-id-instead-of-name)
+        if (parameter.Definition.Name.Equals("Type ID", StringComparison.OrdinalIgnoreCase))
```

Comment:
confirming that localization doesn't impact definition names? is internal name more reliable?

Reply:
ooohpf. good catch! updated in `fix: internal definition name` to use internal name and tested with other language:

<img width="776" alt="image" src="https://github.com/user-attachments/assets/9219e398-20c3-4636-ae61-6bb06ccfa4cc" />